Stratis.Bitcoin.IntegrationTests/RPCTEsts.cs

```diff
@@ -118,5 +123,98 @@ public void CanGetPeersInfo()
                 }
             }
         }
+
+        /// <summary>
+        /// Can call getpeersinfo by string arguments
+        /// </summary>
+        [Fact]
+        public void CanGetPeersInfoByStringArgs()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create())
+            {
+                CoreNode nodeA = builder.CreateStratisPowNode();
+                builder.StartAll();
+                RPCClient rpc = nodeA.CreateRPCClient();
+                using (Node nodeB = nodeA.CreateNodeClient())
+                {
+                    nodeB.VersionHandshake();
+                    var resp = rpc.SendCommand("getpeerinfo").ResultString;
+                    Assert.True(resp.StartsWith("[\r\n  {\r\n    \"id\": 0,\r\n    \"addr\": \"["));
+                }
+            }
+        }
+
+        /// <summary>
+        /// Can GetBlockHash by string arguments
```

Comment:
TBH, I have no idea what this sentence means. How about `Checks that "getblockhash" RPC command returns correct block header hash for regtest genesis block.`

missing dot